src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -103,6 +103,28 @@
             "brightCyan": "#61D6D6",
             "brightWhite": "#F2F2F2"
         },
+        {
+            "name": "Campbell Light",

```

Comment:
we're adding a new stock color scheme? I thought we weren't gonna do that until we got scheme healing. Hmm...

Reply:
Fair point. I thought it made sense, but yea, default schemes annoying.